src/DotNetLightning.Core/Utils/Primitives.fs

```diff
@@ -452,3 +452,41 @@ module Primitives =
         | SortedPlain = 0uy
         | ZLib = 1uy
 
+    type ShutdownScriptPubKey = private {
+        Thingo: Script
```

Comment:
Thingo?

Reply:
Oops. I had a commit to fix that but I forgot to include/squash it.

Reply:
Fixed.

Reply:
is it aussie slang? :D